mails/themes/classic/core/bankwire.html.twig

```diff
@@ -27,7 +27,7 @@
             
 {% endif %}
             <span>
-              {{ 'Your order with the reference <span><strong>{order_name}</strong></span> has been placed successfully and will be <strong>shipped as soon as we receive your payment</strong>.'|trans({}, 'Emails.Body', locale)|raw }}
+              {{ 'Your order with the reference [bold]{order_name}[/bold] has been placed successfully and will be [bold]shipped as soon as we receive your payment[/bold].'|trans({'[bold]': '<span><strong>', '[/bold]': '</strong></span>'}, 'Emails.Body', locale)|raw }}
```

Comment:
I had already removed the html from the keys but using `[bold][/bold]` instead of `[1][/1]` is it ok? or do your wat me to replace them?

Reply:
I think it's best to use `[1]` because that's what we use elsewhere, because it removes the semantics out of the wording, and prevents people from accidentally translating the token.